db/seed.js

```diff
@@ -0,0 +1,13 @@
+const db = require('./mongo.js');
+const data = require('./dataGenerator.js');
+
+
+db.deleteAll();
+
+const seedDB = () => {
+  for (let i = 0; i < data.length; i += 1) {
+    db.save(data[i]);
+  }
+};
+
+seedDB();
```

Comment:
Again not a big deal, but just a note: if you run this from the command line I think it probably won't exit cleanly since you're not closing your db connection. Of course you can always ^C out. But something worth thinking about.

Reply:
thank you, will make an amendment to the code for this suggestion